.github/workflows/ci.yaml

```diff
@@ -31,6 +31,74 @@ jobs:
         with:
           globs: "**/*.md"
 
+  crev:
+    runs-on: ubuntu-22.04
+    permissions:
+      pull-requests: write
+    steps:
+      - uses: actions/checkout@v3
+      - uses: dtolnay/rust-toolchain@stable
+      - uses: Swatinem/rust-cache@v2
+        with:
+          cache-on-failure: "true"
+      - run: cargo check
+      - name: Initialize Cargo Crev
+        run: |-
+          rm -rf ~/.cache/crev
+          rm -rf ~/.config/crev
+          rm -rf ~/.local/share/crev
+          cargo install cargo-crev || true
```

Comment:
:wrench: double checking, the `|| true` is to work around already installed due to `rust-cache`